hs/src/Reach/Connector/ALGO.hs

```diff
@@ -1412,7 +1412,10 @@ ce = \case
   DLE_TimeOrder {} -> impossible "timeorder"
   DLE_GetContract _ -> code "txn" ["ApplicationID"]
   DLE_GetAddress _ -> cContractAddr
-  DLE_EmitLog at _ _ v@(DLVar _ _ _ n)  -> do
+  DLE_EmitLog _ _ _ (L_Event vs) -> do
+    clog $ map DLA_Var vs
```

Comment:
@jeapostrophe  I assume this will need to be tagged with the label, similar to how we tag internal logs with the DLVar int, right? Should we make a map of labels to ints so the tag can be an int and the runtime will know to look at the mapping?